stepsfunction/handler.py

```diff
@@ -361,3 +380,50 @@ def get_s3_resource():
         LOG.info('get_s3_resource initializing new connection')
         S3R = boto3.resource('s3')
     return S3R
+
+
+def ocr_by_tesseract(bucket, key):
+    """Convert PDF to TIF single page, then OCR with tesseract.
```

Comment:
not a big deal, but we're not converting to TIF any longer, just pulling the JPG from the PDF.